docs/config/README.md

```diff
@@ -2009,16 +2009,25 @@ The module will be shown if any of the following conditions are met:
-| `symbol`             | `"🐍 "`                                                                 | A format string representing the symbol of Python                             |
-| `style`              | `"yellow bold"`                                                         | The style for the module.                                                     |
-| `pyenv_version_name` | `false`                                                                 | Use pyenv to get Python version                                               |
-| `pyenv_prefix`       | `pyenv `                                                                | Prefix before pyenv version display, only used if pyenv is used               |
-| `scan_for_pyfiles`   | `true`                                                                  | If false, Python files in the current directory will not show this module.    |
-| `python_binary`      | `python`                                                                | Configures the python binary that Starship executes when getting the version. |
-| `disabled`           | `false`                                                                 | Disables the `python` module.                                                 |
+| Option               | Default                                                                 | Description                                                                            |
+| -------------------- | ----------------------------------------------------------------------- | -------------------------------------------------------------------------------------- |
+| `format`             | `'via [${symbol}${pyenv_prefix}${version}( \($virtualenv\))]($style) '` | The format for the module.                                                             |
+| `symbol`             | `"🐍 "`                                                                 | A format string representing the symbol of Python                                      |
+| `style`              | `"yellow bold"`                                                         | The style for the module.                                                              |
+| `pyenv_version_name` | `false`                                                                 | Use pyenv to get Python version                                                        |
+| `pyenv_prefix`       | `pyenv `                                                                | Prefix before pyenv version display, only used if pyenv is used                        |
+| `scan_for_pyfiles`   | `true`                                                                  | If false, Python files in the current directory will not show this module.             |
+| `python_binary`      | `["python", "python3, "python2"]`                                       | Configures the python binaries that Starship should executes when getting the version. |
```

Comment:
It would be nice to have a small sentence why this default order was chosen: 

```
::: tip

The default values and order for `python_binary` was chosen to first identify the Python version in a virtualenv/conda environments (which currently still add a `python`, no matter if it points to `python3` or `python2`). This has the side effect that if you have still a system Python verison 2 installed, it will be picked up before any Python version 3 (at least on Linux, as Distros do not setup a `/usr/bin/python` symlink for Python 3). If you do not work with Python 2 anymore but cannot remove the system Python version 2, changing this to `["python3"]` will hide any Python version 2. 
```

Reply:
Good suggestion, I've expanded the new tip to include this.